.github/actions/issue-validator/index.js

```diff
@@ -0,0 +1,97 @@
+// @ts-check
+import * as github from '@actions/github'
+import * as core from '@actions/core'
+
+const verifyCanaryLabel = 'please verify canary'
+const bugReportLabel = 'template: bug'
```

Comment:
What about `kind: bug`?

Reply:
That label is added by us when we acknowledged a bug.